src/dialogs.ts

```diff
@@ -124,6 +128,8 @@ export interface IPromptOptions {
   title?: string;
   placeholder?: string;
   multiline?: boolean;
+  primaryBtnText?: string;
+  additionalCSSClasses?: string;
```

Comment:
Maybe you can extract a general options interface and extend `IPromptOptions` (here) and `IChooseOptions` (line 180) from it. (I cannot attach a comment on these lines in the PR review.)

For example:

```ts
interface IDialogOptions {
  title?: string;
  placeholder?: string;
  primaryBtnText?: string;
  additionalCSSClasses?: string;
}

export interface IPromptOptions extends IDialogOptions {
  multiline?: boolean;
}

export interface IChooseOptions extends IDialogOptions {
  editable?: boolean;
}
```

Reply:
thanks for the feedback
done, please check again :+1: